test/suites/nodeclaim/garbage_collection_test.go

```diff
@@ -113,8 +119,8 @@ var _ = Describe("GarbageCollection", func() {
 
 		// Always ensure that we cleanup the instance
 		DeferCleanup(func() {
-			_, err := env.EC2API.TerminateInstances(&ec2.TerminateInstancesInput{
-				InstanceIds: []*string{out.Instances[0].InstanceId},
+			_, err := env.EC2API.TerminateInstances(context.Background(), &ec2.TerminateInstancesInput{
```

Comment:
Is there not a parent context that we want to pass? If not, then we probably want to create this before the function and call `defer ctx.Done()` so we don't leak contexts.